README.md

```diff
@@ -0,0 +1,10 @@
+![](https://runelite.net/img/logo.png)
+# launcher [![Travis](https://img.shields.io/travis/runelite/launcher.svg)](https://travis-ci.org/runelite/launcher) [![Discord](https://img.shields.io/discord/301497432909414422.svg)](https://discord.gg/mePCs8U)
+
+This repository holds the launcher for the runelite client, [click here](https://github.com/runelite/runelite) if you are looking for the runelite client.
+
+If you have any questions, please join our IRC channel on [irc.rizon.net #runelite](http://qchat.rizon.net/?channels=runelite&uio=d4) or alternatively our [Discord](https://discord.gg/mePCs8U) server.
+
+### License
+
+RuneLite is licensed under the BSD 2-clause license. See the license header in the respective file to be sure.
```

Comment:
This also could use some rewording.

```
RuneLite is [licensed under the BSD 2-clause license](https://github.com/runelite/launcher/blob/master/LICENSE).
```

I don't feel it's too important to mention the file license headers.